app/views/provider/admin/authentication_providers/index.html.slim

```diff
@@ -1,29 +1,54 @@
-- content_for(:title)
-  = "Single Sign-on Integrations"
-
-- content_for :page_header_title, "Single Sign-on Integrations"
+- content_for :title, t('.header.title')
+- content_for :page_header_title, t('.header.title')
 
 table class="pf-c-table pf-m-grid-lg" role="grid" aria-label="Authentication providers table"
   thead
     tr role="row"
       th role="columnheader" scope="col" Integration
-      - if branding_relevant?
-        th role="columnheader" scope="col" Branding
       th role="columnheader" scope="col" State
+      th role="columnheader" scope="col" Base URL/Site/Realm
+      td
   tbody role="rowgroup"
-    - for authentication_provider in @authentication_providers
+    - for authentication_provider in presenter.authentication_providers
       tr role="row"
         td role="cell" data-label="Integration" class="provider"
-          - link = "#{icon(authentication_provider.kind, fixed_width: true)} #{authentication_provider.human_kind}".html_safe
+          - link = authentication_provider.human_kind
           - url = authentication_provider.persisted? ? \
               provider_admin_authentication_provider_path(authentication_provider) : \
               new_provider_admin_authentication_provider_path(kind: authentication_provider.kind)
           - if authentication_provider.in_iam_tools_scope?
             = switch_link link, url, switch: authentication_provider.authorization_scope, upgrade_notice: true
           - elsif authentication_provider.in_social_scope?
             = link_to link, url
-        - if branding_relevant?
```

Comment:
I don't really understand what was supposed to be shown here, so I guess it's OK to remove :grimacing: